src/services/update-acsp/updateYourDetailsService.ts

```diff
@@ -0,0 +1,101 @@
+import { ACSPFullProfileDetails } from "../../model/ACSPFullProfileDetails";
+import { Request } from "express";
+import { AcspFullProfile } from "private-api-sdk-node/dist/services/acsp-profile/types";
+import { getFullNameACSPFullProfileDetails } from "../../utils/web";
+
+export const getProfileDetails = (req: Request, acspFullProfile: AcspFullProfile, i18n: any): ACSPFullProfileDetails => {
+    let profileDetails: ACSPFullProfileDetails = {};
+    profileDetails.typeOfBusiness = acspFullProfile.type;
+    profileDetails.correspondenceEmail = acspFullProfile.email;
+    profileDetails.businessName = acspFullProfile.name;
+
+    if (acspFullProfile.type === "limited-company" || acspFullProfile.type === "limited-liability-partnership" || acspFullProfile.type === "corporate-body") {
```

Comment:
Add "limited-company", "limited-liability-partnership" and "corporate-body" to constants file please.